templates/Template App MINIO.xml

```diff
@@ -0,0 +1,294 @@
+                                    <type>JSONPATH</type>
+                                    <parameters>
+                                        <parameter>$.offline_disks</parameter>
+                                    </parameters>
+                                </step>
+                            </preprocessing>
+                            <master_item>
+                                <key>minio.instance-data[{#MINIO_NAME}]</key>
+                            </master_item>
+                            <trigger_prototypes>
+                                <trigger_prototype>
+                                    <uuid>02327db5c3ba469bb635cf4506890088</uuid>
+                                    <expression>last(/Template App MINIO/minio.instance-offline-disks[{#MINIO_NAME}])&gt;0</expression>
+                                    <name>MINIO instance {#MINIO_NAME} has offline disks</name>
+                                    <priority>DISASTER</priority>
+                                </trigger_prototype>
+                            </trigger_prototypes>
+                        </item_prototype>
+                        <item_prototype>
+                            <uuid>27ab130cbfc846918ae2e3a10e233ccc</uuid>
+                            <name>MINIO instance {#MINIO_NAME} online disks</name>
+                            <type>DEPENDENT</type>
+                            <key>minio.instance-online-disks[{#MINIO_NAME}]</key>
+                            <delay>0</delay>
+                            <preprocessing>
+                                <step>
+                                    <type>JSONPATH</type>
+                                    <parameters>
+                                        <parameter>$.online_disks</parameter>
+                                    </parameters>
+                                </step>
+                            </preprocessing>
+                            <master_item>
+                                <key>minio.instance-data[{#MINIO_NAME}]</key>
+                            </master_item>
+                        </item_prototype>
+                    </item_prototypes>
+                </discovery_rule>
+                <discovery_rule>
+                    <uuid>c2b21fd47c4140ffbcddaadf8f21df4f</uuid>
+                    <name>discover-pools</name>
+                    <type>ZABBIX_ACTIVE</type>
+                    <key>minio.discover-pools</key>
+                    <delay>1h</delay>
+                    <lifetime>1d</lifetime>
+                    <item_prototypes>
+                        <item_prototype>
+                            <uuid>b795549d796644728d921d77c63394e9</uuid>
+                            <name>MINIO {#MINIO_NAME} pool {#POOL_ID} decommissioning failed bytes</name>
+                            <type>DEPENDENT</type>
+                            <key>minio.data-decommissioning-failed-bytes[{#MINIO_NAME}, {#POOL_ID}]</key>
```

Comment:
We don't do space between zabbix-key arguments by convention. (Also more below.)